app/src/main/java/deakin/gopher/guardian/view/general/PatientProfileAddActivity.java

```diff
@@ -232,13 +232,15 @@ private void PatientRelatedCreat(final String patient_id) {
   }
 
   private void createMedicalDiagnostic(final String patient_id) {
-    final Medical_diagnostic current_medical_diagnostic = new Medical_diagnostic(patient_id, true);
+    final MedicalDiagnostic currentMedicalDiagnostic = new MedicalDiagnostic(patient_id, null,
+            null,null,null,null,null,null,
```

Comment:
If you look at your model, you can see most fields are assigned `null` as a default value. So you shouldn't need to assign null all over the place here. You should be able to do something like `(patientId, current=true`)